django/db/models/options.py

```diff
@@ -354,6 +336,21 @@ def swapped(self):
                     return swapped_for
         return None
 
+    @property
+    def managers(self):
+        managers = []
+        bases = (b for b in self.model.mro() if hasattr(b, '_meta'))
+        for depth, base in enumerate(bases):
+            for manager in base._meta.local_managers:
+                manager = copy.copy(manager)
```

Comment:
Yeah, I completely agree. Hard to see where it would be an issue in practice, but it really bugs me that we're doing this same copying operation independently, for the same manager, in two different places.